<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>26</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 26pt &quot;微软雅黑&quot;</string>
      </property>
      <property name="text">
       <string>欢迎使用学生选课系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>学生选课系统</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>学生管理</string>
    </property>
    <addaction name="stu_login"/>
    <addaction name="separator"/>
    <addaction name="queryStu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>教师管理</string>
    </property>
    <addaction name="separator"/>
    <addaction name="teacher_login"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>课程管理</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Administrator</string>
    </property>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
   <addaction name="menu_3"/>
   <addaction name="menu_6"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addStu">
   <property name="text">
    <string>添加学生</string>
   </property>
  </action>
  <action name="queryStu">
   <property name="text">
    <string>查询学生</string>
   </property>
  </action>
  <action name="addTeacher">
   <property name="text">
    <string>添加教师</string>
   </property>
  </action>
  <action name="queryTeacher">
   <property name="text">
    <string>查询教师</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>添加课程</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>查询课程</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="stu_login">
   <property name="text">
    <string>学生登陆</string>
   </property>
  </action>
  <action name="teacher_login">
   <property name="text">
    <string>教师登陆</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
